From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Tue, 5 Aug 2025 13:17:15 +0200
Subject: fix(hwdb): make module-setup.sh executable

The `module-setup.sh` scripts are shell scripts and should be
executable.

Forwarded: https://github.com/dracut-ng/dracut-ng/pull/1521
---
 modules.d/74hwdb/module-setup.sh | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 modules.d/74hwdb/module-setup.sh

diff --git a/modules.d/74hwdb/module-setup.sh b/modules.d/74hwdb/module-setup.sh
old mode 100644
new mode 100755
